pages/draft/2020-12/release-notes.md

```diff
@@ -196,7 +200,7 @@ Here's how you would covert a schema using `$recursiveRef` to use `$dynamicRef`.
             }
           }
         }
-        // strict-tree schema, guards against misspelled properties
+      // strict-tree schema, guards against misspelled properties
```

Comment:
Looks like there's still an issue with syntax highlighting. Probably is this comment.

Can you change the language specifier to `jsonc` to see if that fixes the issue?